plugin-build/src/test/kotlin/io/sentry/android/gradle/util/SentryAndroidSdkCheckerTest.kt

```diff
@@ -128,19 +123,22 @@ class SentryAndroidSdkCheckerTest {
         project.detectSentryAndroidSdk(
             fixture.configurationName,
             fixture.variantName,
-            fixture.sdkStateHolderProvider
+            fixture.sentryModulesServiceProvider
         )
 
-        assertTrue { fixture.getSdkState() == SentryAndroidSdkState.MISSING }
+        assertTrue {
+            fixture.getModules().size == 1 &&
+                fixture.getModules()["sentry-android-core"] == SemVer()
+        }
         assertTrue {
             fixture.logger.capturedMessage ==
-                "[sentry] Detected sentry-android MISSING for version: unspecified, " +
-                "variant: debug, config: debugRuntimeClasspath"
+                "[sentry] Detected Sentry modules {sentry-android-core=0.0.0} for variant: debug," +
```

Comment:
How? Imagine you have a `data` module in your app, there's no requirement for it to have a version or anything, because it will be just merged into your final jar/aar. The same way sentry could be added to anyone's project, right.

Or did you confuse it with a dependency coming from `mavenLocal`?

Reply:
Yep, I did though about `mavenLocal`, so we can close this 👍